packages/plugin-char/src/Char.ts

```diff
@@ -47,7 +47,12 @@ export class Char<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
         const range = params.context.range;
         const text = params.text;
         const inline = this.editor.plugins.get(Inline);
-        const modifiers = inline.getCurrentModifiers(range);
+        let modifiers = inline.getCurrentModifiers(range);
+
+        const previousSibling = range.start.previousSibling();
+        if (!previousSibling) {
```

Comment:
I was confused by this for a moment (even though I knew what the feature was about) so I suggest to add a comment on the line before `const previousSibling`:
```// Ony preserved modifiers are applied at the start of a container.```